src/main/java/com/alkemy/ong/mapper/UserMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.alkemy.ong.mapper;
+
+import com.alkemy.ong.domain.User;
+import com.alkemy.ong.repository.model.UserModel;
+
+public class UserMapper {
+    public static User mapModelToDomain(UserModel userModel){
+        User userDomain = User.builder().
```

Comment:
Tratemos de poner el punto al principio del renglón, ejemplo:
```suggestion
        User userDomain = User.builder()
        .firstName(...)
        .lastName(...)
        .build()
```